src/benchmark/CMakeLists.txt

```diff
@@ -26,22 +26,6 @@ target_link_libraries(
     benchmark
```

Comment:
Can't reply to https://github.com/hyrise/hyrise/pull/882#discussion_r202936214 so will do here:

Yes, changed it to your suggestion